src/Nox.ClientApi.Tests/Tests/Controllers/StoreOwnersControllerTests.cs

```diff
@@ -94,5 +94,40 @@ public async Task Post_VatNumberIsCreated()
             queryResult.Should().NotBeNull();
             queryResult!.VatNumber!.Number.Should().Be(expectedVatNumber);
         }
+
+        [Fact]
+        public async Task Post_StreetAddressIsCreated()
```

Comment:
I think we can add two additional tests:

1 - Asserting that we have an exception if we do not fill for example the Country, or required field
2 - Asserting it succeeds if only mandatory fields are set

```suggestion
        public async Task Post_StreetAddressAllFieldsSet_IsCreated()
```

Reply:
Additional tests were added.